nsis/src/nsis/format/NsisCommonHeader.java

```diff
@@ -36,6 +63,42 @@
 				"bgfont block header", "bgfont block header");
 		STRUCTURE.add(NsisBlockHeader.STRUCTURE, NsisBlockHeader.STRUCTURE.getLength(),
```

Comment:
does it imply that there will be only 1 block header here?

Reply:
No there is actually 8 block headers. Is it possible you didn't expand the lines? L48-L64 adds 8 block headers to the structure, each one with a unique name

Reply:
Thanks exactly it, I didn't notice the folded code. 
I'll go sit for a bit in my shame cube 🤦‍♀️